DashSync/Models/Platform/Transitions/BlockchainIdentity/DSBlockchainIdentityCloseTransition.h

```diff
@@ -0,0 +1,22 @@
+//
```

Comment:
Looks like unused file

Reply:
Yep, unused for now, but will use it later.